modular_skyrat/modules/aesthetics/flag/code/signs_flags.dm

```diff
@@ -0,0 +1,160 @@
+	icon_state = "flag_tizira"
+	item_flag = /obj/item/sign/flag/tizira
+
+/obj/structure/sign/flag/mothic
+	name = "flag of the Grand Nomad Fleet"
+	desc = "The flag of the Mothic Grand Nomad Fleet. A classic naval ensign, its use has superceded the old national flag which can be seen in its canton."
+	icon_state = "flag_mothic"
+	item_flag = /obj/item/sign/flag/mothic
+
+/obj/structure/sign/flag/mars
+	name = "flag of the Teshari League for Self-Determination"
+	desc = "The flag of the Teshari League for Self-Determination. Originally a revolutionary flag during the time of the Republic of the Golden Feather, it has since been adopted as the official flag of the planet, as a reminder of how Teshari fought for representation and independence."
+	icon_state = "flag_mars"
+	item_flag = /obj/item/sign/flag/mars
+
+/obj/structure/sign/flag/terragov
+	name = "flag of Sol Federation"
+	desc = "The flag of Sol Federation. It's a symbol of humanity no matter where they go, or how much they wish it wasn't."
+	icon_state = "flag_solfed"
+	item_flag = /obj/item/sign/flag/terragov
+
+/obj/structure/sign/flag/nri
+	name = "flag of the Novaya Rossiyskaya Imperiya"
+	desc = "The flag of the Novaya Rossiyskaya Imperiya. The yellow, black and white colours represent its sovereignity, spirituality and pureness."
+	icon_state = "flag_nri"
+	item_flag = /obj/item/sign/flag/nri
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/flag/nri, 32)
+
+/// Please, for the love of God, use this in Black Mesa ONLY. NOWHERE ELSE. It's the only reason it was allowed in the game.
+/obj/structure/sign/flag/usa
+	name = "flag of the United States of America"
+	desc = "'Stars and Stripes', the flag of the United States of America. Its red color represents endurance and valor; blue one shows diligence, vigilance and justice, and the white one signs at pureness. Its thirteen red-and-white stripes show the initial thirteen founding colonies, and fifty stars designate the current fifty states."
+	icon_state = "flag_usa"
+	item_flag = /obj/item/sign/flag/usa
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/flag/usa, 32)
+
+/obj/structure/sign/flag/syndicate
+	name = "flag of the Syndicate"
+	desc = "The flag of the Sothran Syndicate. Previously used by the Sothran people as a way of declaring opposition against the Nanotrasen, now it became an intergalactic symbol of the same, yet way more skewed purpose, as more groups of interest have joined the rebellion's side for their own gain."
+	icon_state = "flag_syndi"
+	item_flag = /obj/item/sign/flag/syndicate
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/flag/syndicate, 32)
+
+/obj/item/sign/flag
+	name = "folded blank flag"
+	desc = "The folded flag of nothing. It has nothing on it. Beautiful."
+	icon = 'modular_skyrat/modules/aesthetics/flag/icons/flags.dmi'
+	icon_state = "folded_coder"
+	sign_path = /obj/structure/sign/flag
+	is_editable = FALSE
+
+/obj/item/sign/flag/Initialize(mapload)
+	. = ..()
+	var/matrix/Matrix = matrix()
```

Comment:
Just add a comment here explaining why you're doing this. Also name it like `rotation_reset` instead of using capitals in your variable name.

Reply:
Done.